app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  before_save :update_comments_counter
```

Comment:
- [ ] Just to make sure that we increment the counter after a successful save, we should use the `after_save` helper method, so I would suggest you do this insterad:
```suggestion
  after_save :update_comments_counter
```